packit/config.py

```diff
@@ -86,6 +86,8 @@ def __init__(self):
         self._pagure_fork_token: str = ""
         self.dry_run: bool = False
 
+        self.github_requests_log_path: str = ""
```

Comment:
If this is supposed to be set in a config file, then it needs some [docs](https://github.com/packit-service/packit/blob/master/docs/configuration.md#values-1) I think.

Reply:
This would be only used by us for testing purposes, I don't want it to be available to anyone. I can document it in the CONTRIB.md